apps/voting/app/src/components/VoteDetail/VoteInfoBoxes.js

```diff
@@ -0,0 +1,193 @@
+              `}
+            >
+              (>{round(supportRequired * 100, 2)}% needed)
+            </span>
+          </div>
+          <SummaryBar
+            positiveSize={votesYeaVotersSize}
+            requiredSize={supportRequired}
+            css={`
+              margin-top: ${2 * GU}px;
+            `}
+          />
+        </Box>
+      </div>
+      <div>
+        <Box
+          heading={
+            <React.Fragment>
+              Minimum Approval %
+              <Help hint="What is Minimum Approval?">
+                <strong>Minimum Approval</strong> is the percentage of the total
+                token supply that is required to vote “Yes” on a proposal before
+                it can be approved. For example, if the “Minimum Approval” is
+                set to 20%, then more than 20% of the outstanding token supply
+                must vote “Yes” on a proposal for it to pass.
+              </Help>
+            </React.Fragment>
+          }
+          css={`
+            height: 100%;
+          `}
+        >
+          <div
+            css={`
+              ${textStyle('body2')};
+            `}
+          >
+            {round(quorumProgress * 100, 2)}%{' '}
+            <span
+              css={`
+                color: ${theme.surfaceContentSecondary};
+              `}
+            >
+              (>{round(minAcceptQuorum * 100, 2)}% needed)
+            </span>
+          </div>
+          <SummaryBar
+            positiveSize={quorumProgress}
+            requiredSize={minAcceptQuorum}
+            css={`
+              margin-top: ${2 * GU}px;
+            `}
+          />
+        </Box>
+      </div>
+    </div>
+  )
+}
+
+function Status({ vote }) {
+  const theme = useTheme()
+  const network = useNetwork()
+  const { endDate, executionDate, executionTransaction, open } = vote.data
+
+  if (open) {
+    return (
+      <React.Fragment>
+        <div
+          css={`
+            ${textStyle('body2')};
+            color: ${theme.surfaceContentSecondary};
+            margin-bottom: ${1 * GU}px;
+          `}
+        >
+          Time remaining
+        </div>
+        <Timer end={endDate} maxUnits={4} />
+      </React.Fragment>
+    )
+  }
+
+  return (
+    <React.Fragment>
+      <VoteStatus vote={vote} />
+      <div
+        css={`
+          margin-top: ${1 * GU}px;
+          display: inline-grid;
+          grid-template-columns: auto auto;
+          grid-gap: ${1 * GU}px;
+          align-items: center;
+          color: ${theme.surfaceContentSecondary};
+          ${textStyle('body2')};
+        `}
+      >
+        <IconTime size="small" /> {formatDate(executionDate || endDate)}
+      </div>
+      {executionTransaction && (
+        <div>
+          <TransactionBadge
+            networkType={network && network.type}
+            transaction={executionTransaction}
+            css={`
+              margin-top: ${1 * GU}px;
+            `}
+          />
+        </div>
```

Comment:
The `margin-top` doesn't seem to be being applied to `TransactionBadge`, perhaps because the component doesn't support passing them down yet. In the meantime maybe we could move it to the outer `<div>`

Reply:
Good catch!